setup.py

```diff
@@ -25,10 +27,29 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(os.path.abspath(__file__)), fname)).read()
 
+def get_version():
+    version_source = "convert2rhel/__init__.py"
+    with open(version_source) as f:
+        try:
+            return re.findall(
+                r'^__version__ = "([^"]+)"$',
+                f.readline(),
+                re.M,
+            )[0]
+        except IndexError:
+            raise ValueError(
+                (
+                    "Unable to extract the version from %s file. Make sure the "
+                    "first line has the following form: `__version__ = "
+                    '"some.version.here"`'
+                )
+                % version_source
+            )
+
```

Comment:
You do not loop through the file, it will fail if the first line is not `__version__`

Reply:
well, that was intentional. However, I can read the whole file. It is anyway one time overhead

Reply:
fixed